src/olympia/api/tests/test_authentication.py

```diff
@@ -207,29 +209,6 @@ def test_success(self):
         user, _ = self._authenticate(token)
         assert user == self.user
 
-    def test_refresh(self):
```

Comment:
Perhaps this test should stay in a similar fashion to the API key based refresh test?